io/net/bonding.py

```diff
@@ -520,6 +526,16 @@ def test_cleanup(self):
             except Exception:
                 self.log.info("backup file not availbale, could not restore file.")
 
+        detected_distro = distro.detect()
+        if detected_distro.name == "rhel":
+            cmd = "systemctl restart NetworkManager.service"
+        elif detected_distro.name == "Ubuntu":
+            cmd = "systemctl restart networking"
+        else:
+            cmd = "systemctl restart network"
+        if process.system(cmd, shell=True, ignore_status=True) != 0:
+            self.fail("Failed to restart the network service")
+
```

Comment:
Can we have the network restart called even in the peer lpar in the test cleanup ?

Reply:
Done